src/test/munit/2xx/2xx-test.xml

```diff
@@ -0,0 +1,231 @@
+      </http:request>
+    </munit:execution>
+    <munit:validation>
+      <munit-tools:assert-that expression="#[output application/java --- attributes.headers.'Content-type']" is="#[MunitTools::equalTo('application/json')]"/>
+      <munit-tools:assert-that expression="#[output application/java --- attributes.statusCode]" is="#[MunitTools::equalTo(200)]"/>
+      <munit-tools:assert-that expression="#[output application/java --- payload.d.results.OrderID[0]]" is="#[MunitTools::equalTo(10248)]"/>
+      <munit-tools:assert-that expression="#[output application/java --- payload.d.results.OrderID[1]]" is="#[MunitTools::equalTo(10249)]"/>
+    </munit:validation>
+  </munit:test>
+
+  <munit:test name="count-orders">
+    <munit:enable-flow-sources>
+      <munit:enable-flow-source value="2xx-server-main"/>
+      <munit:enable-flow-source value="get:\orders:2xx-server-config"/>
+    </munit:enable-flow-sources>
+    <munit:execution>
+      <http:request method="GET" config-ref="http-requester-simple" path="api/odata.svc/orders/$count">
+        <http:headers>#[output application/java --- {"Accept" : "text/plain"}]</http:headers>
+      </http:request>
+    </munit:execution>
+    <munit:validation>
+      <munit-tools:assert-that expression="#[output application/java --- payload]" is="#[MunitTools::equalTo('2')]"/>
+    </munit:validation>
+  </munit:test>
+
+  <munit:test name="count-orders-without-accept-header">
+    <munit:enable-flow-sources>
+      <munit:enable-flow-source value="2xx-server-main"/>
+      <munit:enable-flow-source value="get:\orders:2xx-server-config"/>
+    </munit:enable-flow-sources>
+    <munit:execution>
+      <http:request method="GET" config-ref="http-requester-simple" path="api/odata.svc/orders/$count">
+      </http:request>
+    </munit:execution>
+    <munit:validation>
+      <munit-tools:assert-that expression="#[output application/java --- attributes.headers.'Content-type']" is="#[MunitTools::equalTo('text/plain')]"/>
+      <munit-tools:assert-that expression="#[output application/java --- payload]" is="#[MunitTools::equalTo('2')]"/>
+    </munit:validation>
+  </munit:test>
+
+  <munit:test name="get-json-order-by-id">
+    <munit:enable-flow-sources>
+      <munit:enable-flow-source value="2xx-server-main"/>
+      <munit:enable-flow-source value="get:\orders\(orderId):2xx-server-config"/>
+    </munit:enable-flow-sources>
+    <munit:execution>
+      <http:request method="GET" config-ref="http-requester-simple" path="api/odata.svc/orders(10248)?$format=json">
+        <http:headers>#[output application/java --- {"Accept" : "application/json"}]</http:headers>
+      </http:request>
+    </munit:execution>
+    <munit:validation>
+      <munit-tools:assert-that expression="#[output application/java --- payload.d.results.OrderID[0]]" is="#[MunitTools::equalTo(10248)]"/>
+    </munit:validation>
+  </munit:test>
+
+  <munit:test name="get-json-order-by-id-without-a-field">
```

Comment:
same, better test name, maybe (as example) "perform-get-request-with-json-body-without-a-REQUIRED-field" or something like that